streams/src/test/java/org/apache/kafka/streams/StreamsBuilderTest.java

```diff
@@ -470,6 +479,233 @@ public void shouldUseSpecifiedNameForSinkProcessor() {
         assertSpecifiedNameForOperation(topology, "KSTREAM-SOURCE-0000000000", expected, "KSTREAM-SINK-0000000001");
     }
 
+    @Test
+    public void shouldUseSpecifiedNameForMapOperation() {
```

Comment:
Do we consider refactoring these tests with one comment function? I could see L485-487 are repeated multiple times.